GNUmakefile

```diff
@@ -0,0 +1,42 @@
+# This GNU make Makefile is intended as a helper.  meson is used for
+# the actual build and configuration, but this GNUmakefile provides
+# some useful targets for development and release purposes.  Type
+# "gmake help" for more details.  It is called GNUmakefile to indicate
+# it requires GNU make.
+
+# Allow calling environment overrides to commands used in this
+# GNUmakefile.  Defaults are below.
+
+REALPATH ?= realpath
+
+# Core directories
+MESON_BUILD_DIR = build
+topdir := $(shell $(REALPATH) $(dir $(lastword $(MAKEFILE_LIST))))
+
+# Project information (may be an easier way to get this from meson)
+PROJECT_NAME = $(shell grep ^project $(topdir)/meson.build | cut -d "'" -f 2)
+PROJECT_VERSION = $(shell grep version $(topdir)/meson.build | grep -E ',$$' | cut -d "'" -f 2)
+
+# Take additional command line argument as a positional parameter for
+# the GNUmakefile target
+TARGET_ARG = `arg="$(filter-out $@,$(MAKECMDGOALS))" && echo $${arg:-${1}}`
+
+all:
+	@echo "Type 'gmake help' for targets."
+
+authors:
+	echo "Authors" > AUTHORS.md
+	echo "=======" >> AUTHORS.md
+	echo >> AUTHORS.md
+	git log --pretty="%aN <%aE>" | sort -u | sed -e 's|^|- |g' | sed G >> AUTHORS.md
```

Comment:
I would set a locale for the sort command, to get a stable order. Different people have different locales.
I would also shrink the to two sed commands into a single `sed -e 's|^|- |g;G'` command.

Reply:
In the future, you should think about markdown-escaping the user name. People can have any characters in their names, especially markdown controll sequences.

Reply:
> In the future, you should think about markdown-escaping the user name. People can have any characters in their names, especially markdown controll sequences.

I'm not opposed to this, but I don't know how to easily do it.  Do you know of an example anywhere?